client/dashboard/task-list/tasks/payments/bacs.js

```diff
@@ -0,0 +1,202 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Component, Fragment } from '@wordpress/element';
+import { Button } from '@wordpress/components';
+import { compose } from '@wordpress/compose';
+import { withDispatch } from '@wordpress/data';
+
+/**
+ * WooCommerce dependencies
+ */
+import { Form, H, TextControl } from '@woocommerce/components';
+
+/**
+ * Internal dependencies
+ */
+import withSelect from 'wc-api/with-select';
+
+class PayFast extends Component {
+	getInitialConfigValues = () => {
+		return {
+			account_name: '',
+			account_number: '',
+			bank_name: '',
+			sort_code: '',
+			iban: '',
+			bic: '',
+		};
+	};
+
+	validate = ( values ) => {
+		const errors = {};
+
+		if ( ! values.account_number && ! values.iban ) {
```

Comment:
Need a sanity check here.  This requires that either an account number or an IBAN is entered, which I believe is the minimum required across all countries.

There is no validation in core at the moment, so we could optionally also remove this validation.